### PR TITLE
Make cuff default range again

### PR DESCRIPTION
**Pull request by Plykiya**
## About the PR

Reverts the nerf from 0.3f back to default range for cuffing

## Why / Balance

"just make it default range" -sloth
https://discord.com/channels/310555209753690112/310555209753690112/1247354757220143144

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

https://github.com/space-wizards/space-station-14/assets/58439124/a72bb67c-d6ea-477f-9d4d-96de17f17b87

**Changelog**

:cl:
- tweak: Handcuff range is buffed to one tile of distance.

### CODE REVIEW

File: Content.Shared/Cuffs/SharedCuffableSystem.cs [hunk 1]
```diff
@@ -484,7 +484,6 @@ public bool TryCuffing(EntityUid user, EntityUid target, EntityUid handcuff, Han
                 BreakOnWeightlessMove = false,
                 BreakOnDamage = true,
                 NeedHand = true,
-                DistanceThreshold = 0.3f
```
Comment (keronshb) [inline Content.Shared/Cuffs/SharedCuffableSystem.cs:487]: This is one tile away this is insane distance for cuffing

**Review by keronshb — CHANGES_REQUESTED** [review]

File: Content.Shared/Cuffs/SharedCuffableSystem.cs (hunk 1 above)
Comment (Plykiya) [reply Content.Shared/Cuffs/SharedCuffableSystem.cs:487]: alright, how about one

**Comment by Plykiya** [comment]
https://github.com/space-wizards/space-station-14/assets/58439124/9113ecbd-c53d-4b9a-bff5-bce996c53e87

1f of distance

File: Content.Shared/Cuffs/SharedCuffableSystem.cs (hunk 1 above)
Comment (metalgearsloth) [reply Content.Shared/Cuffs/SharedCuffableSystem.cs:487]: It's already breakonmove why does this need to be non-default.

Comment (keronshb) [reply Content.Shared/Cuffs/SharedCuffableSystem.cs:487]: 1's fine, general interaction is pretty long and it feels weird cuffing someone from so far away

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**